gravitee-apim-rest-api/gravitee-apim-rest-api-management/gravitee-apim-rest-api-management-rest/src/main/java/io/gravitee/rest/api/management/rest/resource/v4/api/ApiResource.java

```diff
@@ -312,12 +315,12 @@ private void checkApiLifeCycle(ApiEntity api, LifecycleAction action) {
         }
         switch (api.getState()) {
             case STARTED:
-                if (LifecycleAction.START.equals(action)) {
+                if (START.equals(action)) {
                     throw new BadRequestException("API is already started");
                 }
                 break;
             case STOPPED:
-                if (LifecycleAction.STOP.equals(action)) {
+                if (STOP.equals(action)) {
                     throw new BadRequestException("API is already stopped");
                 }
```

Comment:
we could simplify like this:
```
if (STARTED.equals(api.getState()) && START.equals(action)) {
    throw new BadRequestException("API is already started");
}
if (STOPPED.equals(api.getState()) && STOP.equals(action)) {
    throw new BadRequestException("API is already stopped");
}

```

Reply:
It is not only about throwning exception in case action is start or stop but in case of stop there are extra steps.

Reply:
ohhh I see ^^ I should have expand lines